packages/core-flows/src/stock-location/workflows/create-location-fulfillment-set.ts

```diff
@@ -0,0 +1,33 @@
+import { CreateLocationFulfillmentSetWorkflowInputDTO } from "@medusajs/types"
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { createFulfillmentSets } from "../../fulfillment"
+import { associateFulfillmentSetsWithLocationStep } from "../steps/associate-locations-with-fulfillment-sets"
+
+export const createLocationFulfillmentSetWorkflowId =
+  "create-location-fulfillment-set"
+export const createLocationFulfillmentSetWorkflow = createWorkflow(
+  createLocationFulfillmentSetWorkflowId,
+  (input: WorkflowData<CreateLocationFulfillmentSetWorkflowInputDTO>) => {
+    const fulfillmentSet = createFulfillmentSets([
+      {
+        name: input.fulfillment_set_data.name,
+        type: input.fulfillment_set_data.type,
+      },
+    ])
+
+    const links = transform({ input, fulfillmentSet }, (data) => [
```

Comment:
suggestion: I would do that in the `associateFulfillmentSetsWithLocationStep` instead of here to have it wrapped in the step

Reply:
So that someone can reuse the step. the step could accept the same thing as the set region payment providers step

Reply:
Hmm, can I get you to explain what you would like to see changed? The input type wouldn't change, as far as I can see. 

And in the create region workflow, we are also using a transformer to normalize the data for the step 🤔

Reply:
Yes, but we don't create the links shape outside of the step, the step takes care of creating the links so you dont have to worry about how to build link